babel.config.js

```diff
@@ -10,23 +10,49 @@ function normalize(src) {
 module.exports = function (api) {
   const env = api.env();
 
+  const sources = ["packages/*/src", "codemods/*/src", "eslint/*/src"];
+
   const includeCoverage = process.env.BABEL_COVERAGE === "true";
 
-  const envOptsNoTargets = {
-    loose: true,
+  const envOpts = {
     shippedProposals: true,
     modules: false,
     exclude: [
+      "transform-typeof-symbol",
```

Comment:
In `loose` mode `preset-env` disables this plugin, but we don't have an assumption for it because it was done at the preset level. We might consider adding one in the future.

Reply:
Ah, maybe good to note in some places if people miss it when turning off loose mode. although I guess if you are doing this you'd probably want to actually do a code diff to see the changes. Would be nice if we had a tool to help with that